src/systems/filecoin_blockchain/storage_power_consensus/storage_power_actor.md

```diff
@@ -14,58 +14,69 @@ title: Storage Power Actor
 
 {{< readfile file="/docs/actors/builtin/storage_power/storage_power_actor_code.go" code="true" lang="go" >}}
 
+{{<label storage_power>}}
+## On Power
 
-{{<label power_table>}}
-# The Power Table
+Claimed power is assigned to every sector as a static function of its `SectorStorageWeightDesc` which includes `SectorSize`, `Duration`, and `DealWeight`. DealWeight is a measure that maps size and duration of active deals in a sector during its lifetime to its impact on power and reward distribution. A CommittedCapacity Sector will have a DealWeight of zero but all sectors have an explicit Duration which is defined from the ChainEpoch that the sector comes online in a ProveCommit message to the Expiration ChainEpoch of the sector. In principle, power is the number of votes a miner has in leader election and it is a point in time concept of storage. However, the exact function that maps `SectorStorageWeightDesc` to claimed `StoragePower` and `BlockReward` will be announced soon.
 
-The portion of blocks a given miner generates through leader election in EC (and so the block rewards they earn) is proportional to their `Power Fraction` over time. That is, a miner whose storage represents 1% of total storage on the network should mine 1% of blocks on expectation.
+Nominal power is what goes into consensus from claimed power. This includes checks on whether miner's claimed power has met minimum miner size requirement for consensus, whether miner is in DetectedFault state which will result in loss of all power, and whether miner is under collateralized.
+
```

Comment:
> Nominal power is what goes into consensus from claimed power. This includes checks on whether miner's claimed power has met minimum miner size requirement for consensus, whether miner is in DetectedFault state which will result in loss of all power, and whether miner is under collateralized.

I think this is not quite right -- some of the checks don't affect nominal power. More precisely:

- Nominal power = claimed power, unless the miner is in DetectedFault state. Nominal power is used to determine total network storage power for purposes of consensus minimum.
- Consensus power = nominal power, unless the miner fails to meet consensus minimum, or is undercollateralized.